crates/graphql/src/schema/objects/wallet.rs

```diff
@@ -366,6 +366,24 @@ impl Wallet {
             .map_err(Into::into)
     }
 
+    pub fn offers(
+        &self,
+        ctx: &AppContext,
+        offer_type: Option<String>,
```

Comment:
Lets do a graphql enum type here. And then can have a from trait that converts to a string for the query.